Feature/case studies  image grid

k8hansen opened the pull request:
The goal of this card was to replace the placeholder latin data grid with some images, in the interest of launching the Case Studies feature that has been stalled for some time.

Despite the[ Jira card title,](https://b2io.atlassian.net/browse/B2-187) the ultimate solution was not a grid, as the images were not scannable or readable. After consulting with some colleagues, this was the direction we thought would be best to go in while staying away from a light-box.

I did try to add some of the ColorBlock design elements in the back, but it just looked too busy and separate from the _Results_ section.

The images are just in a column on smaller screens, and slightly staggered on larger screens.

I welcome any suggestions, both in design and code! I assume there is a better way to handle these breakpoints, but my brain is stuck on NJ dotcom and that's how we handled it there!

**NOTE:** this is a branch off of the Case Study feature branch.

***

https://user-images.githubusercontent.com/13206464/188969114-8f2fb452-6d94-4e5f-b20a-911c8b62711b.mp4

https://user-images.githubusercontent.com/13206464/188969126-4a72cc29-eed7-4893-adf5-b25e763f372b.mp4

k8hansen commented on src/components/ColorSnapVisualizerPage/ColorSnapImagePanel.tsx:
```diff
@@ -0,0 +1,113 @@
+          width: 85%;
+
+          &:nth-of-type(odd) {
+            right: calc(40% - 50vw) !important;
```
Yes. I used !important. NextJS image seems to be forcing some sort of display properties that I spent hours and hours trying to uncover. This was used elsewhere that had been green-lighted so I did it.

k8hansen replied:
@tborres yes, I actually just came back to say this. It was being used in the Our Team section that Drew made, so I thought it might have been A Thing.

k8hansen replied:
> On my desktop, when I fullscreen this on a 3440X1440, I see the images all the way to the right, like this: ![image](https://user-images.githubusercontent.com/6234996/189768498-02461ca9-2c6b-4aa8-a30d-dd1f7ec170c0.png)
> 
> I think that the -50vw is too much when the view width is very wide

Well that's new! Thanks @zachcd, great catch.

zachcd reviewed (changes requested):
This looks good except for on ultrawide screens the -50vw rule makes the image go way out there.  After that's changed, this looks great!

tborres commented on src/components/ColorSnapVisualizerPage/index.tsx:
```diff
@@ -112,6 +106,7 @@ export const ColorSnapVisualizerPage: FC = () => {
         alt="screenshot"
       />
       <CaseStudyResultsPane {...CaseStudyResultsProps} />
+      <ColorSnapImagePanel alt={''} src={''} />
```
❓ empty string for alt and src?

k8hansen replied:
Making these types optional got rid of this error. That seems not quite right to me, am I missing something?

k8hansen commented:
So in this update I addressed the changes that were out of scope of his PR but needed to be done:

- [x] Added CTA to featured case study on home page
- [x] Added Colorsnap to the nav panel at the bottom. I also rearranged it so the left panel of each page will bring you to the "previous" case study, and the right serves as "next". You should be able to cycle through and see everything this way.

I have no idea why I was trying to align these using positioning, so I switched to flexbox. Everything should look better at larger screens.

Lastly, I fixed the registration mark error, this was likely something I made when I was trying to make it smaller and fix a similar issue on medium screens.

sludemann reviewed (approved):
Looking fantastic on my end. Image changes and content changes all look good.  

Case Study style wise, and not really relevant to this PR so don't fix it here but tablet sizes the description text glues a little uncomfortably close to the left edge of the screen:
Looking fantastic on my end. Image changes and content changes all look good.  
![image](https://user-images.githubusercontent.com/2914538/191814778-cc627ea8-88bf-43c2-ae62-28a41f952062.png)
